cmd/execution_builder.go

```diff
@@ -112,11 +112,6 @@ type ExecutionNode struct {
 	followerState           protocol.MutableState
 	committee               hotstuff.Committee
 	ledgerStorage           *ledger.Ledger
-	events                  *storage.Events
-	serviceEvents           *storage.ServiceEvents
-	txResults               *storage.TransactionResults
-	results                 *storage.ExecutionResults
-	myReceipts              *storage.MyExecutionReceipts
 	providerEngine          *exeprovider.Engine
```

Comment:
@m4ksio are `SeriviceEvents` and `MyExecutionReceipts` specifically for EN? if so i will keep them in execution builder.